### PR TITLE
pullsync: sync only in nn

### PR DESCRIPTION
**Pull request by acud**
#818

### CODE REVIEW

File: pkg/pullsync/mock/pullsync.go [hunk 1]
```diff
@@ -156,15 +157,15 @@ func (p *PullSyncMock) SyncInterval(ctx context.Context, peer swarm.Address, bin
...
+		panic(fmt.Sprintf("bin %d from %d to %d", bin, from, to))
 	}
 
 	if isLive && p.blockLiveSync {
 		// don't respond, wait for quit
 		<-p.quit
 		return 0, 1, context.Canceled
 	}
-
+	fmt.Println(bin, from, to)
```
Comment (janos) [inline pkg/pullsync/mock/pullsync.go:168]: This should be probably removed.

**Review by janos — APPROVED** [review]

**Review by zelig — APPROVED** [review]

File: pkg/pullsync/mock/pullsync.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (zelig) [reply pkg/pullsync/mock/pullsync.go]: outdated

**Comment by Eknir** [comment]
Hi @acud , you closed this PR, but is the issue still relevant? Do we put it in the backlog or in the active sprint backlog?

**Pull request merged**